jetty-documentation/src/main/asciidoc/administration/sessions/session-configuration-infinispan.adoc

```diff
@@ -0,0 +1,77 @@
+//  ========================================================================
+//  Copyright (c) 1995-2016 Mort Bay Consulting Pty. Ltd.
+//  ========================================================================
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+
+[[configuring-sessions-infinispan]]
+
+=== Configuring Inifinspan Clustering
+
+When using the Jetty distribution, you will first need to enable the `session-store-infinispan` link:#startup-modules[module] for your link:#startup-base-and-home[Jetty base] using the `--add-to-start` argument on the command line.
```

Comment:
I split the infinispan stuff into 2 separate modules:    session-store-infinispan-embedded and session-store-infinispan-remote.  And I tidied up the number of jars that are downloaded, so you'll need to update the trace you've cut and pasted in here.

The session-store-infinispan-embedded configures jetty to use an in-process infinispan instance. This can be useful for testing purposes, but in production it is more likely that you will want to use the session-store-infinispan-remote module, which configures jetty to use the HotRod protocol to interact with a remote infinispan server.